contracts/voting-verifier/src/testdata/events_should_not_change.golden

```diff
@@ -1,164 +0,0 @@
-{
-  "event_instantiated": {
-    "attributes": [
-      {
-        "key": "service_name",
-        "value": "serviceName"
-      },
-      {
-        "key": "service_registry_contract",
-        "value": "cosmwasm19d2spdjdycg0tfu27n2zrkwvr25yxwykaxvxwkpnf7hqkka93j4sfp809w"
-      },
-      {
-        "key": "source_gateway_address",
-        "value": "sourceGatewayAddress"
-      },
-      {
-        "key": "voting_threshold",
-        "value": "[\"2\",\"3\"]"
-      },
-      {
-        "key": "block_expiry",
-        "value": "10"
-      },
-      {
-        "key": "confirmation_height",
-        "value": "1"
-      },
-      {
-        "key": "source_chain",
-        "value": "sourcechain"
-      },
-      {
-        "key": "rewards_contract",
-        "value": "cosmwasm1wyhnds94w36ar6jlz4aaaz75cmypwmr0rqnc92nm5m79ejy04hnsx25urh"
-      },
-      {
-        "key": "msg_id_format",
-        "value": "\"hex_tx_hash_and_event_index\""
-      },
-      {
-        "key": "address_format",
-        "value": "\"eip55\""
-      }
-    ],
-    "type": "instantiated"
-  },
-  "event_messages_poll_started": {
```

Comment:
why do we need to remove this? This is concerning because the events should not change. Otherwise we risk breaking external consumers, particularly ampd. For ampd, the messages_poll_started and verifier_set_poll_started should not change.

Reply:
I believe it's just split into multiple different files now